tests/sources/xml/test_ead.py

```diff
@@ -1319,32 +1311,48 @@ def test_get_related_items_success():
             """
             <altformavail>
                 <head>Location of Copies</head>
-                <p>
-            """
-            "A use copy of photographic plates in box 4 can be found "
-            "in the Institute Archives and Special Collections reading room.</p>"
-            """
+                <p>A use copy of photographic plates in box 4.</p>
             </altformavail>
             <separatedmaterial>
                 <head>Separated Materials</head>
                 <p>Issues of Twilight Zine were separated for library cataloging.</p>
             </separatedmaterial>
-            """
```

Comment:
Again, ➕ to dropping the triple quotes, and sorry for missing on prior reviews.  Glad we're establishing some patterns now as we continue into others (though as @ehanson8 pointed out, EAD is probably the most garish XML wise).